### DISCUSSION TITLE
[STEAM 教育學習網]

### DISCUSSION THREAD

**Discussion by WEI-CHEN-CHUN** [1101-GS4719-C/Python-Community-GS4719-C#721] [General]
[STEAM 教育學習網](https://steam.oxxostudio.tw/]
一個很棒的教學平台
可以當作工具書，可以在上面找所需函式或引數

https://steam.oxxostudio.tw/category/python/index.html

**Comment by Yunhans** [comment]
太酷了吧
裏面有超多有趣的內容

**Comment by dinosaur1009** [comment]
真的很實用！適合新手初學者

**Comment by Zoe06060606** [comment]
這裡面的內容好實用！

**Comment by AnAnistired** [comment]
其實類似的教學網站很多，大多都很不錯，但痛點都是廣告有點煩😢

**Comment by williamyen0623** [comment]
第一個網址點開顯示找不到網站？！
